### ISSUE TITLE
Missing point highlights in zone editing in v1.31.39

### ISSUE THREAD

**Issue by xenoxaos** [ZoneMinder/zoneminder#2070]
Two separate installs (I was testing the storage areas PPA on a new VM) and one thing I'm seeing is that the indices for editing a zone are no longer highlighted green in v1.31.39. Is there something I'm missing?
![v1.31.39](https://user-images.githubusercontent.com/543939/38156918-9b951234-3447-11e8-8243-d2932f40ea52.png)
![v1.30.4](https://user-images.githubusercontent.com/543939/38156919-9ba5516c-3447-11e8-8f8f-10332d0dbd59.png)

**Comment by knnniggett** [comment]
@connortechnology Do you know if this still the case, following our recent changes?

**Comment by connortechnology** [comment]
I'm not able to reproduce it, although the last time I saw this it had to do with the css files using ../../../ to get to the images and when you don't have /zm in the url it changes the path.  So I made the decision that no matter how zm is accessed, either with the /zm or without, /zm would still work (by using an alias in apache config).  So now all the images should work.  However it may be a good idea to do some testing.
